conjurapi/authn/api_key_authenticator_test.go

```diff
@@ -47,18 +48,18 @@ func TestAPIKeyAuthenticator_RefreshToken(t *testing.T) {
 
 				token, err := authenticator.RefreshToken()
```

Comment:
Seems like we really only have 2 test cases here:

1. Run RefreshToken() with Login set to "valid-login", and assert good returns
2. Run RefreshToken() with Login set to "invalid-login", and assert that we see expected error.

This can probably done as a table of test cases (with only 2 entries). It would be a little more work, but it would make it very clear what the test case differences are.